Druid/DRMethodIRGenerator.class.st

```diff
@@ -55,6 +56,20 @@ DRMethodIRGenerator >> interpretAssignmentNode: aRBAssignmentNode [
 					  self receiver.
 					  aRBAssignmentNode variable binding index asDRValue.
 					  value } ].
+				
+	aRBAssignmentNode variable binding originalVar isTempVectorTemp ifTrue: [
+		| result |
+		"maybe should create DRStoreVectorTemporaryVariable ???"
```

Comment:
Yes?
Otherwise, how do you differentiate normal stores?

Or for now all stores go to the temp vector?